Backend/app/therapy_sessions/serializers.py

```
@@ -0,0 +1,738 @@
+    """Basic patient information for session displays"""
+    full_name = serializers.ReadOnlyField()
+    patient_id = serializers.SerializerMethodField()
+    
+    class Meta:
+        model = User
+        fields = ['id', 'full_name', 'email', 'phone_number', 'patient_id']
+    
+    def get_patient_id(self, obj):
+        try:
+            return obj.patient_profile.patient_id
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)